src/modules/questionnaire/services/questionnaire.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http'
+import { Observable, throwError, from } from 'rxjs'
+import { catchError } from 'rxjs/operators'
+import { environment } from '../../../environments/environment'
+
+import { IQuestionnaire } from '../../shared/models/questionnaire.model'
+
+const ROUTE_PATH = 'questionnaire/';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
`providedIn:root` is not required since we are lazy loading this service with its module.

Reply:
I tried to remove this but I got some errors I will try to figure out whats is going on... But If you have an idea please let me know.